services/core/shared/core/transactionStatistics.js

```diff
@@ -125,7 +125,10 @@ const insertToDb = async (statsList, date) => {
 
 	statsList.map(statistic => {
 		Object.assign(statistic, { date, amount_range: statistic.range });
-		statistic.id = String(statistic.date).concat('-').concat(statistic.amount_range);
+		statistic.id = String(statistic.date)
+			.concat('-').concat(statistic.type)
+			.concat('-')
+			.concat(statistic.amount_range);
```

Comment:
```suggestion
		statistic.id = `${statistic.date}-${statistic.type}-${statistic.amount_range}`;
```

Reply:
btw, `concat` takes multiple params as well.

Reply:
I updated `concat` to take multiple params

```
statistic.id = String(statistic.date)
		        .concat('-', statistic.type)
			.concat('-', statistic.amount_range);
```